ChemistryLib/PhreeqcIOData/Knobs.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <iosfwd>
+
+namespace ChemistryLib
+{
+namespace PhreeqcIOData
+{
+struct Knobs
+{
+    Knobs(int max_iterations_, double relative_convergence_tolerance_,
```

Comment:
I'd drop the ctor and the `const` from the members.

Reply:
Changed.